spec/lib/submission_builder/ty2024/states/nc/documents/d400_spec.rb

```diff
@@ -13,6 +13,10 @@
     let(:xml) { Nokogiri::XML::Document.parse(build_response.document.to_xml) }
     let(:untaxed_out_of_state_purchases) { "yes" }
 
+    after do
+      expect(build_response.errors).to be_empty
+    end
```

Comment:
since i dont have a fixture ready currently, i wanted to make sure that this would bundle the xml correctly.

Reply:
good point - probably?